src/janus-api/models/feed.ts

```diff
@@ -48,12 +48,12 @@ export class Feed {
   constructor({
     id, display, isPublisher, isLocalScreen, connection, ignored, dataChannelService, eventsService
   }: FeedArgs) {
-    this.id = id;
-    this.display = display;
-    this.publisher = isPublisher;
-    this.localScreen = isLocalScreen;
-    this.ignored = ignored; // is still used?
-    this.connection = connection;
+    this.id = id || 0;
+    this.display = display || null;
+    this.publisher = isPublisher || false;
+    this.localScreen = isLocalScreen || false;
+    this.ignored = ignored || false; // is still used?
+    this.connection = connection || null;
```

Comment:
NP: what about using the [Nullish coalescing operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator) here?